docs/Outbound.md

```diff
@@ -361,3 +361,5 @@ In case you need notes in the new transaction that `send_email()` creates, you s
 ```notes``` option, like so:
 
     outbound.send_email(from, to, contents, outnext, { notes: transaction.notes });
+
+[1]: `Address` objects are address-rfc2821 objects. See https://github.com/haraka/node-address-rfc2821
```

Comment:
Recursive links?

[1]: `Address` objects are instances of [address-rfc2821](https://github.com/haraka/node-address-rfc2821).

source:
`[1]: `Address` objects are instances of [address-rfc2821](https://github.com/haraka/node-address-rfc2821).`